gwsumm/plot/segments.py

```diff
@@ -1376,6 +1384,105 @@ def draw(self, outputfile=None):
 register_plot(SegmentBarPlot)
 
 
+class NetworkDutyBarPlot(SegmentBarPlot):
+
+    """Special case of the `SegmentPiePlot` for network duty factors.
+    """
+    type = 'network-duty-segment-bar'
+    NETWORK_NAME = {
+        0: 'no',
+        1: 'single',
+        2: 'double',
+        3: 'triple',
+        4: 'quadruple',
+        5: 'quintuple',
+        6: 'sextuple',
+    }
+    NETWORK_COLOR = GW_OBSERVATORY_COLORS.copy()
+    defaults = SegmentBarPlot.defaults.copy()
+    # remove SegmentBarPlot default colors as they overwrite
+    # the ones defined later.
+    defaults.pop('color')
+    defaults.pop('edgecolor')
+    defaults.update({
+        'title': 'Network duty factord sad adas ',
```

Comment:
@iaraota Is this a typo?
```suggestion
        'title': 'Network duty factor',
```